### PR TITLE
Print Summary

### PR DESCRIPTION
**Pull request by AnaisHebert**

### CODE REVIEW

File: prime-angular-frontend/src/app/modules/enrolment/pages/summary/summary.component.html [hunk 1]
```diff
@@ -1,7 +1,50 @@
...
+      <dd class="col-sm-12 col-md-6 py-1">{{ enrolment.enrollee.physicalAddress.province }}&nbsp;</dd>
+      <dt class="col-sm-12 col-md-6 py-1">Street Address</dt>
+      <dd class="col-sm-12 col-md-6 py-1">{{ enrolment.enrollee.physicalAddress.street }}&nbsp;</dd>
+      <dt class="col-sm-12 col-md-6 py-1">City</dt>
+      <dd class="col-sm-12 col-md-6 py-1">{{ enrolment.enrollee.physicalAddress.city }}&nbsp;</dd>
+      <dt class="col-sm-12 col-md-6 py-1">Postal Code</dt>
+      <dd class="col-sm-12 col-md-6 py-1">{{ enrolment.enrollee.physicalAddress.postal }}&nbsp;</dd>
+    </dl>
+
+    <h3 class="lead">License Plate</h3>
```
Comment (itchison) [inline prime-angular-frontend/src/app/modules/enrolment/pages/summary/summary.component.html]: licence in Canada.

Comment (itchison) [reply prime-angular-frontend/src/app/modules/enrolment/pages/summary/summary.component.html]: I see a few of these so please find/replace.

File: prime-angular-frontend/src/app/modules/enrolment/pages/summary/summary.component.html [hunk 2]
```diff
@@ -1,7 +1,50 @@
...
+      <dt class="col-sm-12 col-md-6 py-1">Country</dt>
+      <dd class="col-sm-12 col-md-6 py-1">{{ enrolment.enrollee.physicalAddress.country }}&nbsp;</dd>
+      <dt class="col-sm-12 col-md-6 py-1">Province</dt>
+      <dd class="col-sm-12 col-md-6 py-1">{{ enrolment.enrollee.physicalAddress.province }}&nbsp;</dd>
+      <dt class="col-sm-12 col-md-6 py-1">Street Address</dt>
+      <dd class="col-sm-12 col-md-6 py-1">{{ enrolment.enrollee.physicalAddress.street }}&nbsp;</dd>
+      <dt class="col-sm-12 col-md-6 py-1">City</dt>
+      <dd class="col-sm-12 col-md-6 py-1">{{ enrolment.enrollee.physicalAddress.city }}&nbsp;</dd>
+      <dt class="col-sm-12 col-md-6 py-1">Postal Code</dt>
+      <dd class="col-sm-12 col-md-6 py-1">{{ enrolment.enrollee.physicalAddress.postal }}&nbsp;</dd>
```
Comment (mardiel) [inline prime-angular-frontend/src/app/modules/enrolment/pages/summary/summary.component.html]: Probably want to add the postal pipe to this that Martin has in PR49.

Comment (mtpultz) [reply prime-angular-frontend/src/app/modules/enrolment/pages/summary/summary.component.html]: @mardiel I'll add that to my PR once this is merged

File: prime-angular-frontend/src/app/modules/enrolment/pages/summary/summary.component.html [hunk 3]
```diff
@@ -1,7 +1,50 @@
-<div>
+<div class="app-print-layout">
```
Comment (mtpultz) [inline prime-angular-frontend/src/app/modules/enrolment/pages/summary/summary.component.html]: The `app-print-layout` class is no longer used in the application

Comment (AnaisHebert) [reply prime-angular-frontend/src/app/modules/enrolment/pages/summary/summary.component.html]: okay i will remove this

**Pull request merged**